### PR TITLE
Add explicit steps for building the release

### PR DESCRIPTION
**Pull request by starbelly**
- add step to instruct user to cd to their new rebar3 project
 - add step to instruct user to build the release before running it

### CODE REVIEW

**Comment by starbelly** [comment]
@oubiwann I'm tempted to suggest removing the known issue section about ping/pong (i.e., the release not starting up). I tried my best to replicate this but could not. I have a feeling it might have been related to another issue that has been squashed?

File: src/command-ref/projects/release.md [hunk 1]
```diff
@@ -22,6 +22,18 @@ rebar3 new lfe-release prod-lfe
 ===> Writing prod-lfe/config/vm.args
 ```
 
+Change directoy into your new app: 
+
+```shell
+cd prod-lfe
```
Comment (starbelly) [inline src/command-ref/projects/release.md:28]: We may want to change from `prod-lfe` to `prod_lfe` as rebar3 warns about the former: 

```shell
===> The 'name' variable is often associated with Erlang module names and/or file names. The value submitted (prod-lfe) isn't an unquoted Erlang atom. Templates generated may contain errors.
```

Comment (oubiwann) [reply src/command-ref/projects/release.md:28]: Hrm ... Lispers do love their hyphens.

But you bring up a good point; we should definitely at the very least let the reader know they  can  ignore this message (and why!).

**Comment by oubiwann** [comment]
Thanks, @starbelly!

**Pull request merged**